policy/modules/contrib/rtorrent.te

```diff
@@ -0,0 +1,95 @@
+policy_module(rtorrent, 1.0.1)
+
+########################################
+#
+# Declarations
+#
+## <desc>
+## <p>
+## Allow rtorrent to use send mails
+## </p>
+## </desc>
+gen_tunable(rtorrent_send_mails, false)
+
+## <desc>
+## <p>
+## Enable necessary permissions for rutorrent
+## </p>
+## </desc>
+gen_tunable(rtorrent_enable_rutorrent, false)
+
+attribute rtorrentdomain;
+
+attribute_role rtorrent_roles;
+roleattribute system_r rtorrent_roles;
```

Comment:
You should not directly reference `system_r` here. In which scenario is `rtorrent` run by "the system"?